pages/linux/vncviewer.md

```diff
@@ -0,0 +1,19 @@
+# vncviewer
+
+> Launches a VNC (Virtual Network Computing) client.
+
+- Launch a VNC client which connects to a host on a given display:
+
+`vncviewer {{host}}:{{display_number}}`
```

Comment:
From where does one get the display_number ?

Reply:
@agnivade It is the display that the VNC server is running on. I erred away from explaining this concept due to the line in the contributing guidelines "...and avoid explaining general UNIX concepts...". I also decided to keep this consistent with vncserver.md, which had been submitted by someone else already, and which refers to the display_number without definition. ie:

- Kill an instance of VNC Server running on a specific display:
`vncserver --kill :{{display_number}}`

Perhaps I should reword to "Launch a VNC client which connects to a host which is serving on a given display", to make it clear? Any ideas?

Reply:
Yes, I saw the vncserver.md page. Now that I think a bit more, I think this should be fine. I initially thought this to be some vnc specific concept. But if it is a general UNIX concept, we can let it be.

Thanks for the clarification.